### DISCUSSION TITLE
How can I use SeleniumLibrary with this plugin?

### DISCUSSION THREAD

**Discussion by ghilainm** [qualersoft/robotframework-gradle-plugin#70] [Q&A] [answered]
How can I use SeleniumLibrary with this plugin?

**Comment by mathze** [comment, marked as answer]
Hi @ghilainm,

you can add a `robot` dependency to the library like this
```
dependencies {
  robot(group="com.github.marketsquare", name="robotframework-seleniumlibrary", version="4.0.0.0")
}
```
And thank you for remembering me to also document this part of the plugin^^

**Reply by ghilainm** [reply]
Thanks a lot for your feedback. I will try it right away. 

Indeed some recommendation would be really helpful.

Does the plug-in Install Python behind the scene?

Would be nice to have some More details about The Plug in architecture In the documentation.

**Reply by mathze** [reply]
Oh wait... Python?... Are you trying to use the python selenium-library? If so, than the approach will not work (gradle can not natively resolve python libraries). 
As mentioned in the "Attention"-Box, this plugin is a "wrapper for the robotframework standalone-jar". And primarily addresses java-libraries.
If you want to use python libraries, I'm sorry to have to say that this is not handled by the plugin.
In this case you have to find a way to download the python module (maybe there is a python-plugin) and than you can add it to the `additionalPythonPaths` property.
